pkg/plugin/cmd.go

```diff
@@ -196,13 +228,37 @@ func (o *DebugOptions) Validate() error {
 	return nil
 }
 
+// Run run
 func (o *DebugOptions) Run() error {
-
 	pod, err := o.CoreClient.Pods(o.Namespace).Get(o.PodName, v1.GetOptions{})
 	if err != nil {
 		return err
 	}
+	agents, err := o.CoreClient.Pods(v1.NamespaceAll).List(v1.ListOptions{
+		LabelSelector: "app=debug-agent",
+	})
+	if err != nil {
+		return err
+	}
 
+	// get the agent pod which is in the same node with specify pod
+	var agent *corev1.Pod
+	for _, a := range agents.Items {
+		if a.Spec.NodeName == pod.Spec.NodeName {
+			agent = &a
+			break
+		}
+	}
+	if agent == nil {
+		return fmt.Errorf("there is no agent pod in the same node with your speficy pod %s", o.PodName)
+	}
+	fmt.Printf("pod %s PodIP %s, agentPodIP %s\n", o.PodName, pod.Status.PodIP, agent.Status.HostIP)
+	err = o.runPortForward(agent)
+	if err != nil {
+		return err
+	}
+	// wait for port-forward finished
```

Comment:
prefer having a log here (tell user there's something they may have to wait)

Reply:
done